### PR TITLE
BUG: Print an error when no streamlines are created.

### PR DESCRIPTION
**Pull request by danlipsa**

### CODE REVIEW

**Comment by danlipsa** [comment]
@doutriaux1 @aashish24 Please review

File: vcs/vcsvtk/streamlinepipeline.py [hunk 1]
```diff
@@ -128,10 +128,15 @@ def _plotInternal(self):
...
-        numberofglyphsoneside = self._gm.numberofglyphs / 2
-        for i in range(1, numberofglyphsoneside):
-            contour.SetValue(i, r[1] / numberofglyphsoneside * i)
+        if (streamlines.GetNumberOfPoints()):
+            r = streamlines.GetPointData().GetArray("arc_length").GetRange()
+            numberofglyphsoneside = self._gm.numberofglyphs / 2
+            for i in range(1, numberofglyphsoneside):
+                contour.SetValue(i, r[1] / numberofglyphsoneside * i)
+        else:
+            print ("WARNING: No streamlines created. "
```
Comment (doutriaux1) [inline vcs/vcsvtk/streamlinepipeline.py]: @danlipsa can you make this an actual warning?

Comment (danlipsa) [inline vcs/vcsvtk/streamlinepipeline.py]: Done.

**Comment by doutriaux1** [comment]
Thanks. As soon as PR comes back clean I'll merge.

**Comment by danlipsa** [comment]
Do you know what is going on with circleci? Are those differences because of the new way of building mesa? Should I update them?

**Comment by doutriaux1** [comment]
Thanks @danlipsa! I meant to do it for way too long now 😉

**Pull request merged**